R-Interface/jasprcpp.cpp

```diff
@@ -159,23 +159,30 @@ void STDCALL jaspRCPP_init(const char* buildYear, const char* version, RBridgeCa
 	jaspResults::setBaseCitation(baseCitation);
 	jaspResults::setInsideJASP();
 
-	jaspRCPP_logString("Initializing jaspResultsModule, jaspBase, jaspGraphs and more.\n");
+	jaspRCPP_logString("Initializing jaspResultsModule.\n");
 
 	rInside["jaspResultsModule"]			= givejaspResultsModule();
 
 	//Adding some functions in R to the RefClass (generator) in the module
 	//To do: move this entirely to zzzWrapper if this wasn't done yet.
 	jaspRCPP_parseEvalQNT("jaspResultsModule$jaspTable$methods(addColumnInfo = function(name=NULL, title=NULL, overtitle=NULL, type=NULL, format=NULL, combine=NULL) { addColumnInfoHelper(name, title, type, format, combine, overtitle) })");
 	jaspRCPP_parseEvalQNT("jaspResultsModule$jaspTable$methods(addFootnote =   function(message='', symbol=NULL, col_names=NULL, row_names=NULL) { addFootnoteHelper(message, symbol, col_names, row_names) })");
-
-	jaspRCPP_parseEvalQNT("suppressPackageStartupMessages(library(\"jaspBase\"))");
-	jaspRCPP_parseEvalQNT("suppressPackageStartupMessages(library(\"jaspGraphs\"))");
-	jaspRCPP_parseEvalQNT("suppressPackageStartupMessages(library(\"methods\"))");
-	jaspRCPP_parseEvalQNT("suppressPackageStartupMessages(library(\"modules\"))");
+	
+	jaspRCPP_logString("Initializing jaspBase.\n");
+	jaspRCPP_parseEvalQNT("library(\"jaspBase\")");
+	
+	jaspRCPP_logString("Initializing methods.\n");
+	jaspRCPP_parseEvalQNT("library(\"methods\"))");
+	
+	jaspRCPP_logString("Initializing modules.\n");
+	jaspRCPP_parseEvalQNT("library(\"modules\"))");
```

Comment:
Yeah well, that code was there before. So I left it, but you are right, they should just do that (and apparently the `library("modules")` doesnt do it anymore because I had to add `modules::` anyway. 

The `methods` has been there for ever. I can remove it, but I don't honestly know why it is there in the first place.

Reply:
let's remove it, if we run into errors I think we can always put it back.